assets/js/src/core/modules/element/editor/shared-tab-manager/tabs/versions/create-version-accordion-item-functions.tsx

```diff
@@ -91,17 +87,18 @@ export const createVersionAccordionItem = ({
           } }
         />
       )}
-      <span className={ 'title' }>{`${t('version.version')} ${version.versionCount} | ${formatDateTime({
-                timestamp: version.date,
-                dateStyle: 'short',
-                timeStyle: 'medium'
-            })} `}</span>
+      <span className={ 'title' }>
+        {`${i18n.t('version.version')} ${version.versionCount} | ${formatDateTime({
+          timestamp: version.date,
+          dateStyle: 'short',
+          timeStyle: 'medium'
+        })}`}</span>
     </div>
   )
 
   const subtitle = (
     <div>
-      <span className={ 'sub-title' }>{`${t('by')} ${version.user?.name ?? ''}`}</span>
+      <span className={ 'sub-title' }>{`${i18n.t('by')} ${version.user?.name ?? ''}`}</span>
```

Comment:
We should still use the useTranslation hook. Otherwise we loose the reactivity from react and would encounter some issues here the moment we implement the switching of the language.

Reply:
good point, i converted everything to components (see: https://github.com/pimcore/studio-ui-bundle/pull/744/commits/62a163d030d228e76f5a705c8086277a9859df40)